backend/ops_api/Pipfile

```diff
@@ -27,6 +29,7 @@ pytest-flask = "==1.2.0"
 black = {extras = ["d"], version = "==22.12.0"}
 flake8-black = "==0.3.6"
 pytest-cov = "*"
```

Comment:
maybe fix versions for * in file here - also you probably don't need to explicitly include "six" here